cfme/fixtures/pytest_selenium.py

```diff
@@ -112,25 +112,37 @@ def elements(o, **kwargs):
 
     Returns: A list of WebElement objects
     """
-    check_visibility = kwargs.pop("check_visibility", True)
-    if hasattr(o, "locate"):
-        els = elements(o.locate(), **kwargs)
-        if check_visibility:
-            return [e for e in els if is_displayed(e)]
-        else:
-            return els
-    elif callable(o):
-        els = elements(o(), **kwargs)
-        if check_visibility:
-            return [e for e in els if is_displayed(e)]
+    if 'tries' in kwargs:
+        tries = kwargs['tries']
+    else:
+        tries = 3
```

Comment:
I already removed this. I thought the issue was there but I already figured out exact reason.